src/components/Modals/Send/views/Address.tsx

```diff
@@ -92,14 +94,27 @@ export const Address = () => {
                 validateAddress: async (rawInput: string) => {
                   const value = rawInput.trim() // trim leading/trailing spaces
                   setIsValidating(true)
+                  setValue(SendFormFields.To, '')
+                  setValue(SendFormFields.VanityAddress, '')
+                  setValue(SendFormFields.VanityAddress, '')
+                  setValue(SendFormFields.CryptoAmount, '')
                   const { assetId } = asset
                   // this does not throw, everything inside is handled
-                  const parseAddressInputArgs = { assetId, chainId, value }
-                  const { address, vanityAddress } = await parseAddressInput(parseAddressInputArgs)
+                  const parseAddressInputWithChainIdArgs = { assetId, chainId, value }
+                  const { amountCryptoPrecision, address, vanityAddress } =
+                    await parseAddressInputWithChainId(parseAddressInputWithChainIdArgs)
                   setIsValidating(false)
                   // set returned values
                   setValue(SendFormFields.To, address)
                   setValue(SendFormFields.VanityAddress, vanityAddress)
+                  if (amountCryptoPrecision) {
```

Comment:
shouldn't this be `amountCryptoHuman`?

Reply:
Discussed this one out, this is indeed a crypto precision amount, though we should eventually support the spec as a follow-up and don't parse precision amounts (Trust) but rather base units (MM and other wallets following the spec properly)